src/openttd.cpp

```diff
@@ -909,11 +909,14 @@ static void MakeNewGameDone()
 {
 	SettingsDisableElrail(_settings_game.vehicle.disable_elrails);
 
+	bool start_ai_companies = _game_mode != GM_MENU && AI::CanStartNew() && AI::GetStartNextTime() == 0;
```

Comment:
Is this really needed? Is a delay of 1 day really terible? All these exception codes are hard to maintain and difficult to track. Do we really want this?
If so, please do add a comment WHY this is here. As now it just is .. without much context.

Reply:
Still no answer.

Reply:
This is outdated, it was wrong.